WordPress/src/main/res/layout/stats_activity.xml

```diff
@@ -2,185 +2,217 @@
-            android:fillViewport="true">
-            <LinearLayout
-                android:layout_width="match_parent"
-                android:layout_height="match_parent"
-                android:layout_marginLeft="@dimen/content_margin"
-                android:layout_marginRight="@dimen/content_margin"
-                android:orientation="vertical">
+        <android.support.design.widget.AppBarLayout
+            android:layout_height="@dimen/toolbar_subtitle_height"
+            android:layout_width="match_parent">
 
-                <LinearLayout
-                    android:id="@+id/stats_insights_fragments_container"
-                    android:layout_width="match_parent"
+            <android.support.v7.widget.Toolbar
+                android:id="@+id/toolbar_filter"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                style="@style/FilteredRecyclerViewFilterContainer"
+                app:layout_scrollFlags="scroll|enterAlways">
+
+                <Spinner
+                    style="@style/FilteredRecyclerViewSpinner.WordPress"
+                    android:id="@+id/filter_spinner"
+                    android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
-                    android:paddingBottom="@dimen/margin_medium"
-                    android:orientation="vertical"
-                    android:visibility="gone">
-
-                    <!-- LATEST POST SUMMARY SECTION -->
-                    <FrameLayout
-                        android:id="@+id/stats_insights_latest_post_summary_container"
-                        android:layout_width="match_parent"
-                        android:layout_height="wrap_content"
-                        android:layout_marginTop="@dimen/margin_large" />
-
-                    <!-- TODAY SECTION -->
-                    <FrameLayout
-                        android:id="@+id/stats_insights_today_container"
-                        android:layout_width="match_parent"
-                        android:layout_height="wrap_content"
-                        android:layout_marginTop="@dimen/margin_large" />
-
-                    <!-- ALL TIME SECTION -->
-                    <FrameLayout
-                        android:id="@+id/stats_insights_all_time_container"
-                        android:layout_width="match_parent"
-                        android:layout_height="wrap_content"
-                        android:layout_marginTop="@dimen/margin_large" />
-
-                    <!-- POPULAR DAY AND TIME SECTION -->
-                    <FrameLayout
-                        android:id="@+id/stats_insights_most_popular_container"
-                        android:layout_width="match_parent"
-                        android:layout_height="wrap_content"
-                        android:layout_marginTop="@dimen/margin_large" />
+                    android:overlapAnchor="false"/>
```

Comment:
The `overlapAnchor` attribute isn't supported until API 21 (Lollipop). What happens on older versions w/o this?